### DISCUSSION TITLE
Synchronizing expo-sqlite with electricsql to update fields

### DISCUSSION THREAD

**Discussion by GustavoSod** [electric-sql/electric#1990] [General]
Is there any example of an expo-sqlite database synced with electric? I'm trying to make an offline first application with expo, expo-sqlite and electricsql

**Comment by KyleAMathews** [comment]
We haven't yet built a syncing library for sqlite. If you're interested in playing around with this problem & perhaps contributing a library — it'll be pretty similar to the one for pglite — https://github.com/electric-sql/pglite/blob/main/packages/pglite-sync/src/index.ts
